manifests/l/LightburnSoftware/Lightburn/1.1.04/LightburnSoftware.Lightburn.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.5-1-22610-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LightburnSoftware.Lightburn
+PackageVersion: 1.1.04
+PackageLocale: en-US
+Publisher: LightBurn Software
+PublisherUrl: https://lightburnsoftware.com/
+PublisherSupportUrl: https://forum.lightburnsoftware.com/
+# PrivacyUrl: 
+# Author: 
+PackageName: Lightburn
```

```suggestion
PackageName: LightBurn
```
This must match Control Panel.